package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "dataset-addon-for-splunk",
+  "version": "1.0.0",
+  "description": "The Security Data Lake Add-On for Splunk provides integration with [DataSet](https://www.dataset.com) and [XDR](https://www.sentinelone.com/platform/xdr-ingestion) by [SentinelOne](https://sentinelone.com). The key functions allow two-way integration: - SPL custom command to query directly from the Splunk UI. - Inputs to index alerts as CIM-compliant, or any user-defined query results. - Alert action to send events from Splunk.",
+  "main": "index.js",
+  "scripts": {},
+  "keywords": [],
```

Comment:
To ensure the same and correct version if used by everyone, including CI, we should also likely specify min required nodejs version here using ``engines`` attribute - https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines

Reply:
Fixed